### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by skavieller** [JurajNyiri/HomeAssistant-Tapo-Control#745] [General]
Thnks for the update

**Comment by eldi0s** [comment]
Yes, what you have invested making it possible to use third party integrations with the cheap Tapo cams is really honorable. I'm very pleased  to say thank you an keep up the good work. Both thumbs up.

**Comment by Jos1958** [comment]
Yesterday I was very happy to find the update that restores the integration with the Tapo camera’s (without going back to an old firmware version). Thanks very much for not giving up in trying to find a solution with the tp-link people. Great job.
By the way I used the Tapo lab option on my iPad (after an update on the Tapo app) to enable the third party integration (with HA) since this option is not available yet in the Android app on my phone.

**Reply by nivv** [reply]
How do you remove a camera that failed migrating? 

Seems that I need to remove the cameras and add them again.

**Comment by kelvinlimtg** [comment]
Thank you for taking so much time and effort to work on this. Appreciate all your work and dedication in doing this for the community.
